files/usr/share/cinnamon/cinnamon-settings/modules/cs_notifications.py

```diff
@@ -62,6 +62,9 @@ def on_module_selected(self):
         switch = GSettingsSwitch(_("Show notifications on the bottom side of the screen"), "org.cinnamon.desktop.notifications", "bottom-notifications")
         settings.add_reveal_row(switch, "org.cinnamon.desktop.notifications", "display-notifications")
 
+        switch = GSettingsSwitch(_("Show non-critical notifications in fullscreen"), "org.cinnamon.desktop.notifications", "display-noncritical-in-fullscreen")
```

Comment:
This is a bit confusing from a user's point of view. Why non-critical notifications... what about critical ones? Shall we have a simple setting for all notifications instead? "Show notifications in fullscreen", simply.